contracts/src/integrations/cert/EigenDACertVerifier.sol

```diff
@@ -16,14 +16,24 @@ import {EigenDATypesV2 as DATypesV2} from "src/core/libraries/v2/EigenDATypesV2.
 import {IEigenDASemVer} from "src/core/interfaces/IEigenDASemVer.sol";
 
 import {EigenDACertVerificationLib as CertLib} from "src/integrations/cert/libraries/EigenDACertVerificationLib.sol";
-
+import {EigenDACertTypes as CT} from "src/integrations/cert/EigenDACertTypes.sol";
+
+/// @title EigenDACertVerifier
+/// @notice Verifies EigenDA certificates
+/// @dev This contract's checkDACert function is designed to be zk provable by risczero's Steel library,
+/// which does not support zk proving reverting calls: https://github.com/risc0/risc0-ethereum/issues/438.
+/// For this reason, we avoid using revert statements and instead return error codes.
+/// The only acceptable reverts are from bugs, such as contract misconfiguration, which require human intervention.
```

Comment:
nit - not true anymore

Reply:
[45f5c15](https://github.com/Layr-Labs/eigenda/pull/1938/commits/45f5c15309febcfeb600b0340d87d2d3a718863b)